weave/trace_server/model_providers/model_providers.json

```diff
@@ -1 +1,1066 @@
\ No newline at end of file
+{
+  "gpt-4": {
```

Comment:
this is a little ugly now, but in the future it will be easier to see what llms were added